sync/cli/awsdatabricks.py

```diff
@@ -63,10 +64,31 @@ def run_job(job_id: str, plan: str, compute: str, project: dict = None):
 @click.option("--project", callback=validate_project)
 def create_prediction(run_id: str, plan: str, compute: str, project: str = None):
     """Create a prediction for a job run"""
-    prediction_response = awsdatabricks.create_prediction_for_run(
-        run_id, plan, compute, project["id"]
-    )
+    prediction_response = awsdatabricks.create_prediction_for_run(run_id, plan, compute, project)
     if prediction := prediction_response.result:
         click.echo(f"Prediction ID: {prediction}")
     else:
         click.echo(f"Failed to create prediction. {prediction_response.error}", err=True)
+
+
+@aws_databricks.command
+@click.argument("cluster-id")
+@click.option(
+    "--plan", type=click.Choice(["Standard", "Premium", "Enterprise"]), default="Standard"
+)
+@click.option(
+    "--compute",
+    type=click.Choice(["All-Purpose Compute", "Jobs Compute", "Jobs Light Compute"]),
+    default="Jobs Compute",
+)
+def get_cluster_report(cluster_id: str, plan: str, compute: str):
```

Comment:
Perhaps, though I suppose it's kind of a semantic argument - since this is `get_cluster_report`, I think passing the `cluster_id` matches the expected behaviour the closest. However, yeah, we could make this `get_run_cluster_report` instead, and have it accept a `run_id`

Reply:
We don't necessarily have to change the command name (get-run-cluster-report is kinda long). I'm just thinking, I know how to get the run ID easily in the Databricks UI, but I don't know how to get the cluster ID off the top of my head. We could put it out and solicit feedback from Pete, but my hunch is get-cluster-report that takes a run ID is gonna be easier to digest. My 2 cents - not asking for a change.

Reply:
No, it's a good point - the Run ID is definitely easier to get from the Databricks UI vs. a cluster ID.